packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -162,10 +168,12 @@ class MessageList extends Widget<Properties> {
 
   _createMessageGroupComponent(items: Message[], userId: string | number | undefined): void {
     const $messageGroup = $('<div>').appendTo(this._$content());
+    const { messageTimestampFormat } = this.option();
 
     const messageGroup = this._createComponent($messageGroup, MessageGroup, {
       items,
       alignment: this._messageGroupAlignment(userId),
+      messageTimestampFormat: messageTimestampFormat ?? 'shorttime',
```

Comment:
Does it require to add '?? 'shorttime'' because default value of messageTimestampFormat is already 'shorttime'

Reply:
https://github.com/DevExpress/DevExtreme/pull/28234/commits/146df9b953cfca2760258090346ce0e5f1173871